NuGet.Config

```diff
@@ -3,5 +3,6 @@
   <packageSources>
     <clear />
     <add key="roslyn-tools" value="https://dotnet.myget.org/F/roslyn-tools/api/v3/index.json" />
+    <add key="nuget.org" value="https://api.nuget.org/v3/index.json" />
```

Comment:
Does this interfere with the byzantine repo-toolset approach to defining these things?

Reply:
Not sure what you mean. This is needed due to https://github.com/Microsoft/msbuild/issues/2982